pkg/controllers/managementuser/rbac/roletemplates/prtb_handler.go

```diff
@@ -265,19 +266,36 @@ func (p *prtbHandler) buildNamespaceBindings(prtb *v3.ProjectRoleTemplateBinding
 	}
 
 	// check if any of the aggregated CR grant updatepsa permission
+	psaCRName := fmt.Sprintf("%s-namespaces-psa", projectName)
```

Comment:
nit: it's best to avoid using `fmt.Sprintf` where a simple string concatenation can do the job
```suggestion
	psaCRName := projectName + "-namespaces-psa"
```